src/SFA.DAS.ApplyService.Web.UnitTests/Controllers/RoatpManagementHierarchyControllerTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.Domain.Apply;
+using SFA.DAS.ApplyService.Session;
+using SFA.DAS.ApplyService.Web.Controllers.Roatp;
+using SFA.DAS.ApplyService.Web.Infrastructure;
+using SFA.DAS.ApplyService.Web.Services;
+using SFA.DAS.ApplyService.Web.ViewModels.Roatp;
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class RoatpManagementHierarchyControllerTests
+    {
+        private  RoatpManagementHierarchyController _controller;
+        private  Mock<ISessionService> _sessionService;
+        private  Mock<IQnaApiClient> _qnaApiClient;
+        private  Mock<ITabularDataRepository> _tabularDataRepository;
+        private  Guid _applicationId;
+
+        [SetUp]
+        public void Before_each_test()
+        {
+            _applicationId = Guid.NewGuid();
+
+            _sessionService = new Mock<ISessionService>();
+            _qnaApiClient = new Mock<IQnaApiClient>();
+            _tabularDataRepository = new Mock<ITabularDataRepository>();
+
+            var signInId = Guid.NewGuid();
+            var givenNames = "Test";
+            var familyName = "User";
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+            {
+                new Claim(ClaimTypes.Name, $"{givenNames} {familyName}"),
+                new Claim(ClaimTypes.NameIdentifier, "1"),
+                new Claim("Email", "test@test.com"),
+                new Claim("sub", signInId.ToString()),
+                new Claim("custom-claim", "example claim value"),
+            }, "mock"));
+            _controller = new RoatpManagementHierarchyController(_sessionService.Object, _qnaApiClient.Object, _tabularDataRepository.Object)
+            {
+                ControllerContext = new ControllerContext()
+                {
+                    HttpContext = new DefaultHttpContext() { User = user }
+                }
+            };
+        }
+
+        [Test]
+        public  void AddManagementHierarchy_shows_AddManagementHierarchy_page()
```

Comment:
Suggest test name `AddManagementHierarchy_Get_ReturnsAddManagementHierarchyPage`  as per convention mentioned [here](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests).

Rename all the tests accordingly.

Reply:
Renamed